pkg/api/event/nonblocking/adminapi_mutation_addgrouptoroles_executed.go

```diff
@@ -10,6 +10,8 @@ const (
 
 type AdminAPIMutationAddGroupToRolesExecutedEventPayload struct {
 	AffectedUserIDs []string `json:"-"`
+	GroupKey        string   `json:"group_key"`
+	RoleKeys        []string `json:"role_keys"`
```

Comment:
I think we should include ids instead keys in the event. Using IDs make investigation easier. Also, the IDs can be used to build links that link back to the role / group details page.